pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -179,42 +179,47 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	)
 
 	Context("tolerations with eviction policies given", func() {
-		var vmi *v1.VirtualMachineInstance
-		var policyMigrate = v1.EvictionStrategyLiveMigrate
-		var policyMigrateIfPossible = v1.EvictionStrategyLiveMigrateIfPossible
-		var policyNone = v1.EvictionStrategyNone
-		var policyExternal = v1.EvictionStrategyExternal
-
 		BeforeEach(func() {
 			enableFeatureGate(deprecation.LiveMigrationGate)
-			vmi = api.NewMinimalVMI("testvmi")
-			vmi.Spec.EvictionStrategy = nil
 		})
 
-		DescribeTable("it should allow", func(policy *v1.EvictionStrategy) {
-			vmi.Spec.EvictionStrategy = policy
-			resp := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec, config)
-			Expect(resp).To(BeEmpty())
+		DescribeTable("it should allow", func(policy v1.EvictionStrategy) {
+			vmi := newBaseVmi(libvmi.WithEvictionStrategy(policy))
+
+			ar, err := newAdmissionReviewForVMICreation(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			resp := vmiCreateAdmitter.Admit(context.Background(), ar)
+
+			Expect(resp.Allowed).To(BeTrue())
```

Comment:
Please consider adding an assertion that `resp.Result.Message` is empty.
Or alternatively, we could assert that the whole response is equal to the expected response:
https://github.com/kubevirt/kubevirt/blob/4dbd00bb050934f10937fd3a02bb8228068ee1fa/pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter_test.go#L108

Reply:
Asserting `Result.Message` yields a nil pointer dereference as `Result` isn't initialized if an admission is allowed.
I can assert that `Result` is in fact nil instead.